scripts/ci/jsonSchemas/message.json

```diff
@@ -32,6 +32,10 @@
       "type": "string",
       "description": "The type of this message"
     },
+    "notify": {
+      "type": "boolean",
+      "description": "Determines whether we trigger a system notification. Default: true"
+    },
```

Comment:
This addresses https://mozilla-hub.atlassian.net/browse/VPN-3044. Please make sure to assign yourself :100:

Reply:
two for one! thanks